### PR TITLE
Andrew medlin/update docs jun2020

### PR DESCRIPTION
**Pull request by medlin01GA**
Fixing large number of issues blocking full generation of documentation.

### CODE REVIEW

**Comment by medlin01GA** [comment]
@zhang01GA Please review asap. Many of the files are just deleted (redundant code), other changes related to making sphinx documentation be generated properly and cleanly.

If you want to preview the results before merging, you will need to change the configuration on readthedocs to build documenttion in branches.

File: seismic/receiver_fn/c_sippl_reference/HK/LN.SNY.rrf_85_85.list [hunk 1]
```diff
@@ -1,559 +0,0 @@
-2001.001_06.57.LN.SNY.BHRRF.SAC   8.63000   0.10  174.718   34.817
```
Comment (zhang01GA) [inline seismic/receiver_fn/c_sippl_reference/HK/LN.SNY.rrf_85_85.list:1]: Where this external package will be stored then?

Comment (medlin01GA) [reply seismic/receiver_fn/c_sippl_reference/HK/LN.SNY.rrf_85_85.list:1]: Alexei and I both have a copy of this on NCI. I we want to keep a copy of it checked in somewhere, then it should be done as a separate github repository.

**Comment by medlin01GA** [comment]
Last step to getting new docs build working is to make readthedocs build process the same as the 'make docs' we perform when building locally. @zhang01GA will investigate fixing this consistency of the docs build process.

**Pull request merged**